libkbfs/init.go

```diff
@@ -365,9 +366,25 @@ func Init(ctx Context, params InitParams, keybaseServiceCn KeybaseServiceCn, onI
 			onInterruptFn()
 		}
 
-		os.Exit(1)
+		close(done)
 	}()
 
+	errCh := make(chan error)
+	go func() {
```

Comment:
Can you please add some comments here about why we need to call `doInit` in another goroutine?